src/trace_processor/rpc/httpd.h

```diff
@@ -29,7 +29,9 @@ class TraceProcessor;
 // The unique_ptr argument is optional. If non-null, the HTTP server will adopt
 // an existing instance with a pre-loaded trace. If null, it will create a new
 // instance when pushing data into the /parse endpoint.
-void RunHttpRPCServer(std::unique_ptr<TraceProcessor>, const std::string&);
+void RunHttpRPCServer(std::unique_ptr<TraceProcessor>,
```

Comment:
These parameters should be named and need some comments. Two unnamed parameters are very mysterious.